lib/super_diff/rspec/object_inspection/inspectors/collection_including.rb

```diff
@@ -13,8 +13,13 @@ def inspection_tree
             SuperDiff::ObjectInspection::InspectionTree.new do
               add_text "#<a collection including ("
 
-              nested do |aliased_matcher|
-                insert_array_inspection_of(aliased_matcher.expecteds)
+              nested do |value|
+                array = if SuperDiff::RSpec.a_collection_including_something?(value)
+                          value.expecteds
+                        else
+                          value.instance_variable_get(:@expected)
```

Comment:
I understand this is a bad practice, but unfortunately, the rspec-mocks matcher didn't have a public interface to access the values.
https://github.com/rspec/rspec-mocks/blob/v3.10.2/lib/rspec/mocks/argument_matchers.rb#L231-L259

Reply:
Makes sense! (I also do this in other places)